src/core/forces.cpp

```diff
@@ -143,6 +143,8 @@ void force_calc(CellStructure &cell_structure) {
   // Must be done here. Forces need to be ghost-communicated
   immersed_boundaries.volume_conservation();
 
+  lb_reset_force_densities();
```

Comment:
@itischler I think I lost track of what we wanted to achieve... why would we want to reset the force densities here?